M4jonez dev

M4jonez opened the pull request.

M4jonez committed:
1_przetwarzanie_i_wizualizacja

M4jonez committed:
2_wstęp_do_uczenia_maszynowego

M4jonez committed:
1_przetwarzanie_i_wizualizacja_v2

Dodana analiza względem kolumny 'Survived'

fidok15 reviewed (approved):
- do pokazania lepiej użyć [classification_report i ](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html)[confusion_matrix](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.confusion_matrix.html)
- użycie random_state seeda tak aby każdemu kto odpali program wychodziły te same dane 
- można było zobaczyć różne typy DummyClassifier aby zobaczyć który najlepiej sobie radzi 
- dobrze, że jest sprawdzenie czy model przypadkiem nie jest przeuczony/ niedouczony 
- losowo trochę dobrane parametry na koniec max_depth=6, min_samples_split=10 lepiej było pokazać na wykresie jak radzą sobie te parametry dla różnych wartości na wykresie i wtedy dobrać odpowiednie 
- i tak jeszcze wracając do Dokładność: 0.99 Precyzja:  1.0 w drzewie tu raczej napewno wystąpił overfitting 
- trafne wnioski

olabarycka reviewed (commented):
Praca jest czytelna, dobrze pokazujesz cały proces i porównanie modeli. Fajnie, że zwróciłaś uwagę na overfitting w drzewach i standaryzację przy SVM.

M4jonez committed:
1_przetwarzanie_i_wizualizacja_v2

Poprawiona wersja 1 zadania - dodane:
- wyciągnięcie tytułów z kolumny imion
- zostawienie i przetworzenie cabin zamiast usuwania

M4jonez committed:
1_przetwarzanie_i_wizualizacja_v3

(to samo co poprzednie v2 tylko dobrze nazwane)

nuklearbanana reviewed (commented):
Dobrze wyciągnięte wnioski. 
Zamiast drzewa decyzyjnego użyj sobie model RandomForest jest skuteczniejszy. 
Składa się z wielu drzew decyzyjnych i zwraca predykcje, którą zwróciła większość z tych drzew.
